.github/workflows/test-notify-slack-workflow.yml

```
@@ -0,0 +1,24 @@
+---
+
+name: 'Run notify-slack workflow'
+
+on:
+  pull_request:
+    paths:
+      - .github/actions/test-notify-slack/*
```

```suggestion
      - .github/workflows/notifty-slack.yml
      - `actions/notify-slack/*`
```

This workflow is meant to test the test-notify-slack workflow.  We do not call the test-notify-slack action at all and so this path is unneeded.  These paths are the paths of files that, when changed, we should run this workflow.  We can also include the `actions/notify-slack/*` path, but will need to make use of the `use-local-actions`.  We can talk about this in Office Hours.